docs/scenarios_authoring/RRDataSourcesAndSinks.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 4
+---
+
+# Request-Response sources and sinks
+
+## Http request as input and output to Nussknacker scenarios
+
+### Prerequisites
+
+To fully understand how Nussknacker works in Request-Response mode, it's best to read the following first:
+* [Http overview](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview)
+
+### Concepts
+
+This section discusses HTTP based source and sink components.
+
+Nussknacker can process single HTTP request as a scenario input. 
+In this case source and input are integrally connected meaning that the source component defines a response for incoming request for exact HTTP session.
```

Comment:
It still can be difficult for non technical user. 

_In Request-response mode you interact with Nussknacker Lite runtime engine in a similar way as with any "server" in the Internet. You send request which contains request parameters and you get response from the server. The protocol for this communication is HTTP - the same protocol is used by browsers when they communicate with web servers. One of the differences is that the content of the communication with the Nussknacker Lite engine ("the server") is encoded as JSON, not HTML.  Not very surprisingly, in Request-response processing mode the only possible Source component is Request and the only Sink component Response._